src/redfish/messages/messages.py

```diff
@@ -0,0 +1,174 @@
+#! /usr/bin/python
+# Copyright Notice:
+# Copyright 2019-2020 DMTF. All rights reserved.
+# License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/Redfish-Tacklebox/blob/main/LICENSE.md
+
+"""
+Messages Module
+
+File : messages.py
+
+Brief : This file contains the definitions and functionalities for interacting
+        with Messages for a given Redfish service
+"""
+import re
+
+class RedfishOperationFailedError( Exception ):
+    """
+    Raised when an operation has failed (HTTP Status >= 400)
+    """
+    pass
+
+class RedfishPasswordChangeRequiredError( Exception ):
+    """
+    Raised when password change required
+    """
+    def __str__(self):
+        return  "\n{}\nURL: {}\n".format( str(self.args[0]), str(self.args[1]) )
+
+def print_error_payload( response ):
```

Comment:
Let's remove "print_error_payload" for now; we don't have a use case for printing details at the lower protocol library, and it might not make sense to encourage others to try to use this.